miso-web/src/main/webapp/scripts/hot_dilution.js

```diff
@@ -199,6 +199,56 @@ HotTarget.dilution = {
 
     HotUtils.makeParents('librarydilution', HotUtils.relationCategoriesForDetailed().concat([HotUtils.relations.library()])), 
     HotUtils.makeChildren('librarydilution',[HotUtils.relations.pool()])];
+  },
+
+  confirmSave: function(flatObjects, isCreate, config, table) {
+    var deferred = jQuery.Deferred();
+    
+    dilutions = table.getDtoData();
+    
+    var seen = {};
+    libraries = dilutions.filter(function(dilution){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "libraries" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)